.github/workflows/ci.yml

```diff
@@ -1,6 +1,8 @@
 name: Brim CI
 
-on: [push]
+on:
+  pull_request:
+  push:
```

Comment:
As I suspected, this triggers separate events for the push and pull_request. The easiest solution is to go to the zq model, which only triggers push events on master.

That means the convenience of getting auto-CI whenever you push now requires you to open a draft.

Reply:
At https://github.com/brimsec/brim/actions/runs/147602102 notice "on: push" on the left.
https://github.com/brimsec/brim/actions/runs/147602683 is for `pull_request`.